gthompson2/martins-aquarium
# automated webpage quotes and tips with JavaScript. Removed hard coded quotes and tips

## Changes Made:
1) Automated webpage quotes and tips with JavaScript 
2) Removed hard coded quotes and tips
## How to Check:
1) Verify that quotes and tips directories exist
2) Check that objects in QuoteDataProvider.js and TipDataProvider.js are being successfully converted to HTML and displayed on the webpage
